plugins/output/flight/flight.go

```diff
@@ -0,0 +1,164 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+// SampleConfig returns the configuration of the Apache Arrow Flight plugin.
+func (*Flight) SampleConfig() string {
+	return sampleConfig
+}
+
+// Flight contains the configuration, data structures, and interfaces for the Apache Arrow Flight plugin.
+// It is the primary data structure for the plugin.
+type Flight struct {
+	Location string `toml:"location"`
+	Port     string `toml:"port"`
+	Table    string `toml:"table"`
+
+	client flight.FlightServiceClient
+	writer flight.Writer
+	schema arrow.Schema
+	ctx    context.Context
+	pool   memory.Allocator
+	desc   *flight.FlightDescriptor
+}
+
+// Write should write immediately to the output, and not buffer writes
+// (Telegraf manages the buffer for you). Returning an error will fail this
+// batch of writes and the entire batch will be retried automatically.
+func (f *Flight) Write(metrics []telegraf.Metric) error {
+
+	builder := array.NewRecordBuilder(f.pool, &f.schema)
+	defer builder.Release()
+
+	//Currently, the plugin only implements support for the simplest schema
```

Comment:
Missing space after // for multiple comments in the PR. Although I do not know if spaces are allow for "//go:embed sample.conf".

Reply:
Fixed.